idaes/generic_models/properties/swco2.py

```diff
@@ -166,21 +164,17 @@ def build(self):
         )
 
         self.tc_d_1 = Param(
-            initialize=2.447164e-2,
-            doc="Residual thermal conductivity parameter",
-            units=pyunits.mW/pyunits.K*pyunits.m**2/pyunits.kg)
+            initialize=2.447164e-5,
+            doc="Residual thermal conductivity parameter")
         self.tc_d_2 = Param(
-            initialize=8.705605e-5,
-            doc="Residual thermal conductivity parameter",
-            units=pyunits.mW/pyunits.K*pyunits.m**5/pyunits.kg**2)
```

Comment:
Should we really be removing units from these? Unless you actually do a convert later, having units here shouldn't slow anything down, and having them allows us to do unit consistency checks which are good for giving people confidence in our code.

Reply:
Agree with you will fix.